services/api-server/requirements/_test.in

```diff
@@ -29,3 +29,4 @@ pytest-runner
 respx
 sqlalchemy[mypy] # adds Mypy / Pep-484 Support for ORM Mappings SEE https://docs.sqlalchemy.org/en/20/orm/extensions/mypy.html
 types-boto3
+async_asgi_testclient # only added to test streaming endpoints (see https://github.com/tiangolo/fastapi/discussions/9126).
```

Comment:
MINOR: please keep alphabetical order in all human-written listings! Our experience shows that we reduce duplicates ( you have still not experience the situation of having one letter different names in packages. lots of fun to maintain `aiofiles` and `aiofile` in a repo with 500+ dependencies ;-)

Reply:
Right, good point. I think I can even get rid of this one again actually. I will check if I am even using it in the last tests I implemented